taca/illumina/HiSeqX_Runs.py

```diff
@@ -287,7 +287,7 @@ def _generate_samplesheet_subset(ssparser, lanes):
         if line['Lane'] in lanes:
             line_ar=[]
             for field in datafields:
-                if field == "index" and "NOINDEX" in line[field]:
+                if field == "index" and ("NOINDEX" in line[field] or "NoIndex" in line[field]):
```

Comment:
If you wanna catch all kinds of combination the following will be good, but if it should only be the above mentioned than it is good 👍 

`if field == "index" and "noindex" in line[field].lower()`

Reply:
Thanks Senthil! I will use the `.upper()` function instead as it is more consistent with the original code.